tools/project_manager/page_compare.php

```diff
@@ -0,0 +1,203 @@
+<?php
+$relPath="./../../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath.'theme.inc');
+include_once($relPath.'Project.inc');
+include_once($relPath."DPage.inc"); // remove_formatting()
+
+require_login();
+
+$L_round_options = array('P1', 'P2', 'P3', 'F1');
+$R_round_options = array('F1', 'F2');
+
+$projectid = validate_projectID('project', @$_GET['project']);
+$L_round_id = get_enumerated_param($_GET, "L_round_id", "P3", $L_round_options);
+$R_round_id = get_enumerated_param($_GET, "R_round_id", "F1", $R_round_options);
+$page_set = get_enumerated_param($_GET, "page_set", "all", array('left', 'right', 'all'));
+$go_compare = isset($_GET['compare']);
+
+$project = new Project($projectid);
+
+$state = $project->state;
+$title = $project->nameofwork;
+$sub_title = _('Compare pages with formatting removed');
+
+output_header("$sub_title: $title", NO_STATSBAR);
+
+echo "<h1>$title</h1>\n";
+echo "<h2>$sub_title</h2>\n";
+
+$project_url = "$code_url/project.php?id=$projectid&amp;expected_state=$state";
+$label = _("Return to Project Page");
+
+echo "<p><a href='$project_url'>$label</a></p>\n";
+
+if (!$project->pages_table_exists)
+{
+    echo "<p>";
```

Comment:
Should these be styled with `class='warning'` or `error`? I could go either way on that.

Reply:
I copied this block of code from "tools/project_manager/page_detail.php" so should we change it there also or perhaps better to make it a function somewhere?